### PR TITLE
Add Content: C++

### PR DESCRIPTION
**Pull request by JDSherbert**
As described.

### CODE REVIEW

File: docs/Language/C++/C++.md [hunk 1]
```diff
@@ -9,9 +9,135 @@ has_children: true
...
+C++ is a general-purpose programming language that was developed as an extension of the C programming language. It was created by Bjarne Stroustrup at Bell Labs in the early 1980s. C++ introduces object-oriented programming features to C, allowing developers to write more organized and modular code.
 
+-----------------------------------------------------------------------
 
-<br>
+### Official Documentation
 
-<br>
\ No newline at end of file
+- Microsoft Documentation: https://learn.microsoft.com/en-us/cpp/cpp/
```
Comment (VerzatileDev) [inline docs/Language/C++/C++.md:18]: Should be made into a clickable link Consider adding [https://learn.microsoft.com/en-us/cpp/cpp/](https://learn.microsoft.com/en-us/cpp/cpp/) Otherwise it remains as a text 'without linking.

Comment (JDSherbert) [reply docs/Language/C++/C++.md]: Fixed

File: docs/Language/C++/C++.md [hunk 2]
```diff
@@ -9,9 +9,135 @@ has_children: true
...
+- CPlusPlus: https://cplusplus.com/
+- CPP Reference: https://en.cppreference.com/w/
+- IsoCPP: https://isocpp.org/std/the-standard
+
+-----------------------------------------------------------------------
+
+### Features
+
+- Object-Oriented Programming (OOP): C++ supports the four main principles of OOP: encapsulation, inheritance, polymorphism, and abstraction. This makes it possible to model real-world entities more effectively.
```
Comment (VerzatileDev) [inline docs/Language/C++/C++.md:27]: Too wordy, consider "Re-tuning" it.   And I believe that rest of the content should be ordered based on  " encapsulation, inheritance, polymorphism, and abstraction " Ordering given.  Personally I would tune it to Talk about what the Terminology means And then what falls into that category.   " Object... Or known as (OOP) " <- Or wise versa. some text ... And then categories like you already listed.

Comment (JDSherbert) [reply docs/Language/C++/C++.md:27]: There is a dedicated OOP section in "Concepts/OOP"

File: docs/Language/C++/C++.md [hunk 3]
```diff
@@ -9,9 +9,135 @@ has_children: true
...
+
+
+
+-----------------------------------------------------------------------
+
+### Getting Started
+
+Using C++ involves several steps, from setting up your development environment to writing, compiling, and running your code. Here's a step-by-step guide to get you started:
+
+1. ### Install a C++ Compiler
```
Comment (VerzatileDev) [inline docs/Language/C++/C++.md:73]: Not sure if this is an issue referenced locally or " 
image
 
The Numbers seem to be showing  1. 1. 1. For some reason.

Comment (JDSherbert) [reply docs/Language/C++/C++.md:73]: Doesn't seem to be an issue locally. 
image

Comment (JDSherbert) [reply docs/Language/C++/C++.md:73]: Note that if you view the Raw file, I've manually typed 1., 2., 3., etc

File: docs/Language/C++/C++.md [hunk 4]
```diff
@@ -9,9 +9,135 @@ has_children: true
...
+
+- C++20: Released in 2020, C++20 included major features such as concepts (which allow template constraints), ranges library, coroutines, and improvements to existing features like constexpr.
+
+Each release has aimed to enhance the language's expressiveness, safety, and performance, while also improving the overall developer experience. As of my last knowledge update in January 2022, C++23 was under development, but I don't have specific details about its features and changes. It's always a good idea to check the latest documentation and releases for the most up-to-date information.
+
+
+
+-----------------------------------------------------------------------
+
+### Getting Started
```
Comment (VerzatileDev) [inline docs/Language/C++/C++.md:69]: Should we Change Sections Starts with  `##` instead of `###`, which splits and identifies the start of the section and inside the section.

**Review by VerzatileDev — APPROVED** [review]
See Mentioned Comments, Otherwise Approved.

File: docs/Language/C++/C++.md (hunk 4 above)
Comment (JDSherbert) [reply docs/Language/C++/C++.md:69]: Up to you, I'm not following any particular pattern. I can change this now?

File: docs/Language/C++/C++.md (hunk 3 above)
Comment (VerzatileDev) [reply docs/Language/C++/C++.md:73]: Might be that github automatically thinks that each one of them is starting a new " Ordered List " and that is why it show as such. Strange issue, have experienced that myself occasionally

File: docs/Language/C++/C++.md (hunk 4 above)
Comment (VerzatileDev) [reply docs/Language/C++/C++.md]: Yeah seems that Underline comes at `#` and not with double. So add section starts with `##` and inside content smaller.

**Comment by JDSherbert** [comment]
Feedback addressed.

**Review by VerzatileDev — APPROVED** [review]
Awesome, well done!
